packages/suite/src/components/suite/layouts/SuiteLayout/useAppShortcuts.tsx

```diff
@@ -29,5 +29,11 @@ export const useAppShortcuts = () => {
             );
             e.preventDefault();
         }
+
+        // press CMD + , to show Settings
+        if (metaKey && e.code === 'Comma' && selectedDevice) {
```

Comment:
future idea: As keybinds are gonna grow, we may consider refactor those IFs into some map.

Reply:
nit: `selectedDevice !== undefined`?